src/core/plugins/oas3/helpers.jsx

```diff
@@ -8,7 +8,7 @@ export function isOAS30(jsSpec) {
 
   return (
     typeof oasVersion === "string" &&
-    /^3\.0\.([0123])(?:-rc[012])?$/.test(oasVersion)
+    /^3\.0\.([01234])(?:-rc[012])?$/.test(oasVersion)
```

Comment:
We need to have this aligned with how [ApiDOM hadles version matching](https://github.com/swagger-api/apidom/blob/main/packages/apidom-parser-adapter-openapi-json-3-0/src/adapter.ts#L12)

We now need to be forward compatible with only supporting sensible future versions.

```suggestion
    typeof oasVersion === "string" && /^3\.0\.(?:[1-9]\d*|0)$/.test(oasVersion)
```